docs/guides/admin/docs/configuration/security.jwt.md

```diff
@@ -102,79 +102,43 @@ An external application wants to let a user use Opencast Studio to record and up
 Spring Security Configuration
 -----------------------------
 
-In order to active JWT-based authentication and authorization, you will need to uncomment and adapt the following
-sections found in `etc/security/mh_default_org.xml`. Some of the options are configured with the
-[Spring Expression Language (SpEL)](https://docs.spring.io/spring-framework/docs/3.0.x/reference/expressions.html).
+In order to active JWT-based authentication and authorization, you will need to adjust your security config (found in `etc/security/mh_default_org.xml`).
 
-* Enable the `preauthAuthProvider`.
-```xml
-<!-- Uncomment this if using Shibboleth or JWT authentication -->
-<sec:authentication-provider ref="preauthAuthProvider" />
-```
-* Enable the `userReferenceProvider`.
-```xml
-<!-- Uncomment to enable external users e.g. used together with shibboleth or JWT -->
-<osgi:reference id="userReferenceProvider" cardinality="1..1"
-                interface="org.opencastproject.userdirectory.api.UserReferenceProvider" />
-```
-* Add the configured `jwtHeaderFilter` and/or `jwtRequestParameterFilter` to the filters list
-  of the `preAuthenticationFilters` bean.
-```xml
-<!-- Uncomment the line below to support JWT. -->
-<ref bean="jwtHeaderFilter" />
-<!-- Additionally/alternatively uncomment this to support passing a JWT in a URL parameter. -->
-<ref bean="jwtRequestParameterFilter" />
-```
-* Configure the `jwtHeaderFilter` and/or `jwtRequestParameterFilter` beans.
-```xml
-<!-- General JWT header extraction filter -->
-<bean id="jwtHeaderFilter" class="org.opencastproject.security.jwt.JWTRequestHeaderAuthenticationFilter">
-  <!-- Name of the HTTP request header that contains the JWT (default: SM_USER) -->
-  <property name="principalRequestHeader" value="Authorization"/>
-  <!-- Prefix string occurring before the JWT value within the configured header (default: null) -->
-  <property name="principalPrefix" value="Bearer "/>
-  <property name="authenticationManager" ref="authenticationManager" />
-  <property name="loginHandler" ref="jwtLoginHandler" />
-  <!-- Throws an exception if a request is missing the configured header (default: true) -->
-  <property name="exceptionIfHeaderMissing" value="false" />
-</bean>
+If you just want to be able to accept JWTs from Tobira or LMS, you only have to fill out a single value: `jwksUrl` (just search for that string, it's part of the `jwtLoginHandler` bean configuration). Just set the corresponding Tobira/LMS URL there (check their docs to find the URL). All other JWT configuration can usually be kept as is, as the defaults work fine for most cases.
```

Comment:
That "Tobira or LMS" is really a "Tobira xor LMS". I haven't looked at how multiple JWKSes can be combined to accept keys from Tobira AND an LMS. Maybe mention this?